src/Sulu/Bundle/AdminBundle/Resources/js/components/Block/ActionPopover.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+import React from 'react';
+import {Menu, Popover} from '..';
+import Icon from '../Icon';
+import actionPopoverStyles from './actionPopover.scss';
+import type {ActionConfig} from './types';
+import type {ElementRef} from 'react';
+
+type Props = {
+    actions: Array<ActionConfig>,
+    anchorElement: ?ElementRef<'*'>,
+    onClose: () => void,
+    open: boolean,
+};
+
+export default class ActionPopover extends React.PureComponent<Props> {
+    render() {
+        const {
+            open,
+            onClose,
+            anchorElement,
+        } = this.props;
+
+        return (
+            <Popover
+                anchorElement={anchorElement}
+                onClose={onClose}
+                open={open}
+                verticalOffset={5}
+            >
+                {(setPopoverRef, popoverStyle) => (
+                    <Menu
+                        menuRef={setPopoverRef}
+                        style={popoverStyle}
+                    >
+                        {this.props.actions.map((action, index) => {
+                            if (action.type === 'divider') {
+                                return <Menu.Divider key={index} />;
+                            }
+
+                            return (
+                                <li key={index}>
```

Comment:
To be consistent I would create a MenuOption and or MenuItem component or what best matching name is. So we also get rid of the `react/jsx-no-bind` ignore here.

Reply:
i extracted a `ActionPopoverItem`. but it really feels cumbersome to do this, i am not a fan of the `react/jsx-no-bind` rule 🙈